nums/core/array/application.py

```diff
@@ -464,9 +468,13 @@ def linspace(self, start, stop, shape, block_shape, endpoint, retstep, dtype, ax
         assert axis == 0
         assert endpoint is True
         assert retstep is False
-        step_size = (stop - start) / (shape[0] - 1)
-        result = self.arange(shape, block_shape)
-        result = start + result * step_size
+
+        result = self.arange(start, shape, block_shape)
```

Comment:
I think if we call `self.arange(0, shape, block_shape)` here, we can retain the original implementation, which invokes fewer nums ops (2 vs 5). Thoughts?

Reply:
Let me give it a shot and see if it works.

Reply:
Can confirm it works.